__tests__/homepage.test.tsx

```diff
@@ -2,9 +2,15 @@ import { render } from "@testing-library/react";
 import Home from "../src/pages/index";
 import { axe } from "jest-axe";
 import { props } from "./data/homepageProps";
+class ResizeObserver {
+  observe() {}
+  unobserve() {}
+  disconnect() {}
+}
 
 describe("Homepage Accessibility", () => {
   it("passes axe accessibility test", async () => {
+    window.ResizeObserver = ResizeObserver;
```

Comment:
Did this give you an error before you added this? Do you remember what it was?

Reply:
`ReferenceError: ResizeObserver is not defined`

Reply:
I encountered this same issue in a DS PR I am working on. It affected multiple tests. Eventually, when new pages are added, this error will most likely come up again.

I did a global fix by adding your code (thanks!) into the DS [`src/__tests__/setup.ts`](https://github.com/NYPL/nypl-design-system/pull/1448/files#diff-a68a2df534784b96b4ce1a9d4d9580bc33679e819076557c07a191d2b0d5dc15) file. You can do the same by moving 
```
class ResizeObserver {
  observe() {}
  unobserve() {}
  disconnect() {}
}

window.ResizeObserver = ResizeObserver;
```

into the repo's `./jest.setup.ts` file.